README.md

```diff
@@ -54,6 +56,10 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## Video Presentaion <a href="https://www.loom.com/share/61f331921b974c76b0bfd2b6e51b8209?sid=5fa316cc-7bad-4a45-be69-aa8115efdfa3" name="live-demo">video-link</a>
+
+## Live Demo <a href="https://my-budget-hq38.onrender.com" name="live-demo-link">Live demo link</a>
+
 <!-- GETTING STARTED -->
```

Comment:
- Hello, can you "acknowledge" or give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).?